### PR TITLE
update

### PR DESCRIPTION
**Pull request by ilovemane**

### CODE REVIEW

**Comment by ilovemane** [comment]
Hey Marti, I have to comment out some test related to summariseIndication. Since we nee to re-code that function I guess?

File: R/addIndication.R [hunk 1]
```diff
@@ -43,64 +44,51 @@
...
-#'     indicationCohortName = "indication_cohorts",
-#'     indicationGap = c(0, 30, 365)
-#'   )
+#' cdm[["drug_cohort"]] |>
+#'   addIndication(cdm, "indication_cohorts", indicationWindow = list(c(0,0)))
 #' }
 #'
 addIndication <- function(x,
-                          cdm = lifecycle::deprecated(),
+                          cdm = attr(x ,"cdm_reference"),
```
Comment (catalamarti) [inline R/addIndication.R:52]: can we eliminate the cdm argument?

File: R/addIndication.R [hunk 2]
```diff
@@ -43,64 +44,51 @@
...
-      ind %>% dplyr::rename(!!indicationDate := "cohort_start_date"),
-      by = c("subject_id", indicationDate)
-    ) %>%
-    dplyr::compute()
-
-  dropTmpTables(cdm = cdm)
+      ind |> dplyr::rename(!!indexDate := "cohort_start_date"),
+      by = c("subject_id", indexDate)
+    ) |> indicationToStrata() |>
+    CDMConnector::computeQuery()
```
Comment (catalamarti) [inline R/addIndication.R:91]: can we use dplyr::compute(name = ..., temporary = ...)? if we need to compute inside the function could we make all the intermediate tables permanents and we delete them?

File: R/addIndication.R [hunk 3]
```diff
@@ -43,64 +44,51 @@
...
                           unknownIndicationTable = NULL,
-                          indicationDate = "cohort_start_date") {
-  if (lifecycle::is_present(cdm)) {
-    lifecycle::deprecate_soft(
-      when = "0.5.0", what = "addIndication(cdm = )"
-    )
-  }
-
-  cdm <- omopgenerics::cdmReference(x)
+                          indexDate = "cohort_start_date") {
```
Comment (catalamarti) [inline R/addIndication.R:57]: could we have a name argument so that we compute the table to where we need it. default name = NULL (temporary table)

**Review by catalamarti — CHANGES_REQUESTED** [review]
hi @ilovemane some small changes that would be needed, thank you very much

**Review by catalamarti — APPROVED** [review]

**Comment by catalamarti** [comment]
thanks @ilovemane

**Pull request merged**